symphony/bdk/core/symphony_bdk.py

```diff
@@ -26,6 +37,20 @@ async def bot_session(self) -> AuthSession:
             self._bot_session = await self._authenticator_factory.get_bot_authenticator().authenticate_bot()
         return self._bot_session
 
+    async def messages(self):
+        if self._message_service is None:
+            self._message_service = MessageService(
+                MultiAttachmentsMessagesApi(self._agent_client),
+                MessageApi(self._pod_client),
+                MessageSuppressionApi(self._pod_client),
+                StreamsApi(self._pod_client),
+                PodApi(self._pod_client),
+                AttachmentsApi(self._agent_client),
+                DefaultApi(self._pod_client),
+                await self.bot_session()
```

Comment:
Regarding the lazy init, I discussed it quickly with @symphony-elias and this works fine for now !!